### DISCUSSION TITLE
Provider "artifactory" not available for installation.

### DISCUSSION THREAD

**Discussion by eric-lopez** [jfrog/terraform-provider-artifactory#442] [Q&A] [answered]
Not able to find the artifactory provider from https://releases.hashicorp.com/ 

` % terraform init

Initializing provider plugins...
- Checking for available provider plugins on https://releases.hashicorp.com...

Provider "artifactory" not available for installation.

A provider named "artifactory" could not be found in the official repository.

This may result from mistyping the provider name, or the given provider may
be a third-party provider that cannot be installed automatically.

In the latter case, the plugin must be installed manually by locating and
downloading a suitable distribution package and placing the plugin's executable
file in the following directory:
    terraform.d/plugins/darwin_amd64

Terraform detects necessary plugins by inspecting the configuration and state.
To view the provider versions requested by each module, run
"terraform providers".`

`% cat main.tf
terraform {
  required_providers {
    artifactory = {
      source = "jfrog/artifactory"
      version = "6.6.0"
    }
  }
}

provider "artifactory" {
  url           = "${var.artifactory_url}/artifactory"
  access_token  = "${var.artifactory_access_token}"
}`

**Comment by alexhung** [comment, marked as answer]
Looks like your `source` attribute has incorrect value.

```hcl
terraform {
  required_providers {
    artifactory = {
      source  = "registry.terraform.io/jfrog/artifactory"
      version = "6.21.3"
    }
  }
}
```
